test/bike_brigade/importers/mailchimp_importer_test.exs

```diff
@@ -0,0 +1,50 @@
+defmodule BikeBrigade.Importers.MailchimpImporterTest do
+  use BikeBrigade.DataCase
+
+  alias BikeBrigade.Importers.MailchimpImporter
+
+  @valid_attrs %{
+    id: "123",
+    email_address: "dispatcher@example.com",
+    status: "subscribed",
+    timestamp_opt: "2019-08-24T14:15:22Z",
+    merge_fields: %{
+      FNAME: "Morty",
+      PHONEYUI_: "647-555-5555",
+      PHONE: "647-555-5555",
+      TEXTYUI_3: "1508 Davenport Rd"
+    }
+  }
+  test "parse_mailchimp_attrs is successful if fields from Mailchimp are valid" do
```

Comment:
`parse_mailchimp_attrs` should probably be untested and a private method.. was keeping it around for debugging and because `sync_riders` handles the errors without returning anything. thoughts welcome!